system/include/libcxx/atomic

```diff
@@ -539,7 +539,8 @@ void atomic_signal_fence(memory_order m) noexcept;
 
 _LIBCPP_BEGIN_NAMESPACE_STD
 
-#if !__has_feature(cxx_atomic) && _GNUC_VER < 407
+// #if !__has_feature(cxx_atomic) && _GNUC_VER < 407 // XXX Emscripten: Added ' && !defined(__EMSCRIPTEN__)'
+#if !__has_feature(cxx_atomic) && _GNUC_VER < 407 && !defined(__EMSCRIPTEN__)
```

Comment:
It's built into clang. In lib/Lex/PPMacroExpansion.cpp in clang, cxx_atomic is set to true when clang is in C++11 mode (-std=c++11 or similar), which is unfortunately not currently on by default.